tests/ci/reporter_tests.js

```diff
@@ -92,38 +95,96 @@ describe('test reporters', function(){
       reporter.finish()
       var output = stream.read().toString()
       assert.match(output, /<testsuite name="Testem Tests" tests="1" failures="0" timestamp="(.+)" time="([0-9]+)">/)
-      assert.match(output, /<testcase name="phantomjs it does &lt;cool&gt; &quot;cool&quot; &apos;cool&apos; stuff"\/>/)
+      assert.match(output, /<testcase classname="phantomjs" name="it does &lt;cool> &quot;cool&quot; \'cool\' stuff"/)
```

Comment:
It won't break anyone, as it's a new attribute, but it will be correct
xunit xml, so parsers will like it better.

Working on a readme change

On 1 April 2015 at 12:12, Johannes Würbach <notifications@github.com> wrote:

> In tests/ci/reporter_tests.js
> <https://github.com/airportyh/testem/pull/530#discussion_r27560579>:
>
> > @@ -92,38 +95,96 @@ describe('test reporters', function(){
> >        reporter.finish()
> >        var output = stream.read().toString()
> >        assert.match(output, /<testsuite name="Testem Tests" tests="1" failures="0" timestamp="(.+)" time="([0-9]+)">/)
> > -      assert.match(output, /<testcase name="phantomjs it does &lt;cool&gt; &quot;cool&quot; &apos;cool&apos; stuff"\/>/)
> > +      assert.match(output, /<testcase classname="phantomjs" name="it does &lt;cool> &quot;cool&quot; \'cool\' stuff"/)
>
> Sorry, I think this is sane.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/airportyh/testem/pull/530/files#r27560579>.
>



-- 
Bryan Crotaz
Managing Director
Silver Curve